includes/settings/js/src/components/fields/AdvancedSettings.jsx

```diff
@@ -1,7 +1,7 @@
 /**
  * Additional field options that appear in the Advanced Settings modal.
  */
-import React from "react";
+import React, { useRef } from "react";
```

Comment:
`useRef` is introduced here but not used.

```suggestion
import React from "react";
```